rusk-profile/src/lib.rs

```diff
@@ -57,7 +57,12 @@ fn file_stem(p: &Path) -> Option<&str> {
 }
 
 pub fn get_rusk_profile_dir() -> Result<PathBuf, io::Error> {
-    if let Some(mut dir) = home_dir() {
+    if let Some(env_var) = option_env!("RUSK_PROFILE_PATH") {
+        let mut path = PathBuf::from(env_var);
+        path.push(".rusk");
+        fs::create_dir_all(&path)?;
+        Ok(path)
+    } else if let Some(mut dir) = home_dir() {
```

Comment:
Unfortunately I cannot select lines that are not part of this PR.
However, you should be able to remove some code duplication having something like:
```rust
pub fn get_rusk_profile_dir() -> Result<PathBuf, io::Error> {
    option_env!("RUSK_PROFILE_PATH")
        .map_or(home_dir(), |e| Some(PathBuf::from(e)))
        .and_then(|mut p| {
            p.push("./rusk");
            fs::create_dir_all(&p).map(|_| p).ok()
        })
        .ok_or_else(|| {
            warn!("rusk-profile dir not found and impossible to create");
            io::Error::new(
                io::ErrorKind::NotFound,
                "User Profile Dir not found",
            )
        })
}
```
Maybe you can even improve that, but it's just to give you the gist.

Reply:
Yupp. That looks much better!!!